ui_framework/components/button/_button.scss

```diff
@@ -18,7 +18,8 @@
   border-radius: $globalBorderRadius;
 
   @include kuiButtonDisabled {
-    cursor: default;
+    cursor: not-allowed;
+    opacity: .5;
```

Comment:
The disabled state doesn't quite look disabled in some contexts:

![image](https://cloud.githubusercontent.com/assets/1238659/25755821/52071d46-3179-11e7-9f04-391261d27d74.png)

Some solutions off the top of my head:

1. Change the context, e.g. make this sidebar have a white background.
2. Change the button enabled state so it contrasts more, e.g. by adding a box-shadow or border. Then the disabled state will look "flatter".
3. Change the button disabled state so it contrasts more, e.g. by also dropping the text opacity and/or by reducing the saturation of the button so it looks a little more grayed out.